### PR TITLE
Feat: change getopt to OptionParser, update readme, add one user mult…

### PR DESCRIPTION
**Pull request by henry-lp**
Hi @gluckzhang :) , I change getOpt to OptionParser to be able to add role to an existing service account user . In exchange for that the syntax changed a little bit .

### CODE REVIEW

File: 9-Tools/configfile_generator/ConstructAccess.py [hunk 1]
```diff
@@ -286,48 +421,34 @@ def case7():
...
-	"createExCustomRole":case5,"recreate":case6,"limit":case7,"help":case8}
-	try:
-		opts, args = getopt.getopt(sys.argv[2:], 'n:u:a:r:l:',['nsname=', 'uname=','akind=','rpath=','lpath='])
-	except getopt.GetoptError:
-		print(bcolors.FAIL + "Input Error" + bcolors.ENDC)
-		sys.exit(2)
-	optsdict = dict(opts)
-	if sys.argv[1] not in cases:
+	cases = {"help":case0,"merge":case1,"create":case2,"createEx":case3,"createCustomRole":case4,
+	"createExCustomRole":case5,"recreate":case6,"limit":case7,"addRoles":case8}
```
Comment (gluckzhang) [inline 9-Tools/configfile_generator/ConstructAccess.py]: Method `caseN` is not very readable, could you please rename them? For example, `case0` could be `help()` instead.

**Review by gluckzhang — COMMENTED** [review]
Thanks @Tailp ! That would be better if method `caseN` could be better readable :)

File: 9-Tools/configfile_generator/ConstructAccess.py (hunk 1 above)
Comment (henry-lp) [reply 9-Tools/configfile_generator/ConstructAccess.py]: how about I call `cases` dictionary as `methods` ? . If I understand correctly the problem is the keys in cases are not cases* . :)

Comment (gluckzhang) [reply 9-Tools/configfile_generator/ConstructAccess.py]: I mean you have a series of methods, like `case0()`, `case1()`, etc. Why not directly name them as `help()`, `merge()`?

Comment (henry-lp) [reply 9-Tools/configfile_generator/ConstructAccess.py]: Ah I see what you mean now :) . I'll see what I can do.

Comment (henry-lp) [reply 9-Tools/configfile_generator/ConstructAccess.py]: It is now refactored :) .

**Comment by gluckzhang** [comment]
Thanks @Tailp , it is merged!

**Pull request merged**